client/src/pages/Home.js

```diff
@@ -36,6 +38,16 @@ export function Home() {
 					{message}
 				</h1>
 				<Link to="/about/this/site">About</Link>
+				<Box>
+				<IconButton
+      component="a"
+      href="https://mui.com/material-ui/react-image-list/"
```

Comment:
It is better to link to a Facebook page like https://www.facebook.com according by the ticket.

Reply:
To my knowledge Jarrod said just any link, as long as it works.